### ISSUE TITLE
code reorg

### ISSUE THREAD

**Issue by rfvander** [epizon-project/Fenix#32]
- Source  files are gradually getting out of hand in terms of size. We need to break them into pieces and give them recognizable names

- Source and header files are currently mixed. all header files should go to the include directory

- We need to clean up stray text, decide on a uniform coding and commenting style

**Comment by keitat** [comment]
Any suggestions?  As for headers, I do not see any in src  except the one auto-generated by .in file.

**Comment by rfvander** [comment]
It was a general comment. It will take some time to develop a uniform coding style. Of course, if you already have one, we can adopt that. 

Yes, can we move that generated header file to /include as well?

**Comment by keitat** [comment]
Yes, I can.  I guess we also want to make the best effort to keep 1-to-1 correspondence between .h and .c files.
